probdists/Triangulardistribution.py

```diff
@@ -0,0 +1,199 @@
+import math
+import matplotlib.pyplot as plt
+from .Generaldistribution import Distribution
+from collections import Counter
+
+
+class Triangular(Distribution):
+    """
+    Triangular distribution class for calculating and visualizing the
+    triangular distribution: a continuous probability distribution shaped
+    like a triangle
+    Note: a <= mode <= b
+
+    Attributes:
+
+        a (float): the minimum lower limit value
+        b (float): the maximum upper limit value
+        mode (float): the mode, where min <= mode <= max
+
+        mean (float): the mean value of the distribution
+        stdev (float): the standard deviation of the distribution
+
+    """
+
+    def __init__(self, a=0, b=1, mode=0.5):
+        if mode < a or b < mode:
```

Comment:
Perhaps a better way of writing this woudl be b < mode < a

Reply:
Fixed